frb_example/benches/dart/lib/constants.dart

```diff
@@ -0,0 +1,13 @@
+const int defaultSampleCount = 50;
+
+/// in milliseconds
```

Comment:
I don't think `Duration` is available as a primitive for env, but I need to double-check. Otherwise there's the trick to allow users specifying their duration in human readable format, e.g. like with [humantime](https://docs.rs/humantime/latest/humantime/), but I don't know yet if such is available in Dart. Let me have a look later on if you don't mind.

Reply:
The humantime is not needed (but if you like surely you can :) ) i.e. how users specify the time

What I think most important is, we should let *users* of this variable see type Duration instead of int, otherwise they can easily use it wrongly by interpreting it as microsecond.

Maybe something like `var yourTime = Duration(milliseconds: readTheTimeInMilliseconds)`.

> Let me have a look later on if you don't mind.

surely take your time!